packages/block-library/src/post-featured-image/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-featured-image` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-featured-image` block on the server.
+ *
+ * @return string Returns the featured image for the current post.
+ */
+function render_block_core_post_featured_image() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	return '<p>' . get_the_post_thumbnail( $post ) . '</p>';
```

Comment:
Semantically, this would be better wrapped with a `figure` tag , which should additionally have a class name so that it can be targeted by Theme and Plugin developers.